src/fides/api/api/v1/endpoints/health.py

```diff
@@ -84,7 +84,7 @@ async def health(
     db: Session = Depends(get_db),
```

Comment:
Is this parameter still needed now that the `get_db_health` call doesn't have `db=db` in it?